packages/Uploader/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@paprika/uploader",
+  "version": "0.1.0",
+  "description": "Uploader component for paprika library",
+  "author": "@paprika",
+  "main": "lib/index.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika/tree/master/packages/"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
+    "@paprika/helpers": "^0.1.2",
+    "@paprika/l10n": "^0.2.1",
+    "@paprika/raw-button": "^0.1.6",
+    "@paprika/stylers": "^0.1.5",
+    "@paprika/tokens": "^0.1.4",
+    "prop-types": "^15.7.2",
+    "uuid": "^3.3.3"
+  },
+  "peerDependencies": {
+    "react": "^16.8.4",
+    "react-dom": "^16.8.4",
+    "styled-components": "^4.2.0",
+    "superagent": "^5.1.0"
```

Comment:
Why peer? Seems like a regular dependency to me.

Reply:
what if a consumer has already declare superagent? I trying to avoid similar conflicts that we could run into like the one with `numeraljs` been declare in one app and the need to use a different version in our sriracha library. 

That's why I did it.

Reply:
(I don't remember/know what it was but) It's possible that `numeraljs` had an issue that required only having one version of it. But unless there is such evidence, we should go with a regular dependency and let the package manager sort out the versions. It would also not be nice to require someone to manually install superagent if they were not already using it.